src/android/SMSRetriever.java

```diff
@@ -152,7 +153,12 @@ public void onSuccess(Void aVoid) {
                     intentFilter.addAction(SmsRetriever.SMS_RETRIEVED_ACTION);
 
                     try {
-                        applicationContext.registerReceiver(smsBroadcastReceiver, intentFilter);
+                        if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.TIRAMISU) {
```

Comment:
we will have error if the targetSDK is lower then TIRAMISU(33)

error: cannot find symbol if (Build.VERSION.SDK_INT < Build.VERSION_CODES.TIRAMISU)
Build.VERSION_CODES.TIRAMISU only exists in API 33,

please change  to 
```java
ANDROID_VERSION_TIRAMISU = 33;
     if (Build.VERSION.SDK_INT >= ANDROID_VERSION_TIRAMISU) {
applicationContext.registerReceiver(smsBroadcastReceiver, intentFilter,
                                    Context.RECEIVER_EXPORTED);
        } else {
                            applicationContext.registerReceiver(smsBroadcastReceiver, intentFilter);
                                    }
```